feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -63,6 +64,7 @@ export class AddToCartComponent implements OnInit, OnDestroy {
   @ViewChild('addToCartDialogTriggerEl') addToCartDialogTriggerEl: ElementRef;
 
   maxQuantity: number;
+  realTimeStock: string = '';
```

Comment:
MAJOR: can we refactor the code here? something like - reuse `maxQuantity` property itself and populate it with real time stock in case the feature is ON. So that later in `getInventory` method, we will not need extra logic.
We can also remove feature toggle check inside `refreshLiveStockData` and invoke this method from `setStockInfo` based on feature toggle

Reply:
Done 
Although we need to call refreshLiveStockData under nginit else it will result in infinite api calls